check_line_length.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import glob
+import os
+import re
+import sys
+
+LIMIT = 75
+
+
+def Find(string):
```

Comment:
Hi,
Please change:
`Find` to something like `check_url` avoid using capital letters unless you are naming a class. 

Also, `string` might not be the best variable name. 
In fact, i try to avoid any generic name while naming an argument or variable.
Self descriptive names help someone understand the code and also avoid any conflicts with standard keywords or imported variables.

Reply:
Agreed. I would definitely try to give more descriptive names from next time.. it's just that till now i was more focused on learning the language so its a bit habitual to give generic names but definitely making it clear is as imp as anything.